### PR TITLE
docs: add tldr version to quickstart

### PR DESCRIPTION
**Pull request by leeederek**
## Description:

- Adds a dropdown tab for the TLDR version of the quickstart, instructing users to simply run the quickstart package and read the "review" sections if theyre curious how things got setup
- Removes the tip box about "copy-paste-able" code blocks. That should be obvious i think and expected. 
- Removes the feedback dropdown tab, in favor of keeping a single dropdown tab in the introduction

## Is this change user facing?
YES

### CODE REVIEW

**Comment by leeederek** [comment]
@galenmarchetti  thoughts?

**Comment by leeederek** [comment]
image

@galenmarchetti  thoughts?

File: docs/docs/quickstart.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ slug: /quickstart
 Introduction
 ------------
 
+<details><summary>TL;DR Version</summary>
+
+This quickstart is in a "code along" format. If you'd prefer to dive straight into the code and inner workings of the distributed app we build here, clone the [`awesome-kurtosis` repository][awesome-kurtosis-repo], get [set up](#setup), and then run:
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
This quickstart is in a "code along" format. If you'd prefer to dive straight into running the end results and exploring the code, [install Kurtosis](#setup), clone the [`awesome-kurtosis` repository][awesome-kurtosis-repo], and then run:
```

File: docs/docs/quickstart.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ slug: /quickstart
...
+
+```bash
+cd awesome-kurtosis/quickstart && kurtosis run --enclave quickstart main.star 
+```
+
+Once you've got the app up and running, `cd` into [`go-test`][go-test-example] to run a simple Go unit test against your app. Refer to the [recap](#conclusion) and the other `Review` sections in this quickstart guide to learn more about what we did and how we did it.
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
Once you've got your test environment up and running, `cd` into [`go-test`][go-test-example] to run a simple Go unit test against your app. Refer to the [recap](#conclusion) and the other `Review` sections in this quickstart guide to learn more about what we did and how we did it.
```

**Review by galenmarchetti — CHANGES_REQUESTED** [review]

**Review by galenmarchetti — APPROVED** [review]

**Pull request merged**